bin/www

```diff
@@ -1,9 +1,13 @@
 // This will be our application entry. We'll setup our server here.
 const http = require('http');
+const db = require('../server/models/index');
 const app = require('../app'); // The express app we just created
 
-const port = parseInt(process.env.PORT, 10) || 8000;
+const port = process.env.PORT || 8000;
```

Comment:
I felt it might just turn out to be showboating...I don't think it is important....I stand to be corrected though....If you advise otherwise I will adhere.